backend/api/Areas/Admin/Controllers/ParcelController.cs

```diff
@@ -103,17 +117,20 @@ public IActionResult GetParcelByPid(int pid)
         /// </summary>
         /// <param name="pid">The unique 'PID' for the parcel to return.</param>
         /// <returns>The parcel requested.</returns>
-        [HttpGet("pid/{pid}")]
+        [HttpGet("p-id/{pid:pid}")]
```

Comment:
The route above this one is for ` [HttpGet("pid/{pid:int}")]`, however only this one got renamed to `[HttpGet("p-id/{pid:pid}")]` with a "-". Could you please provide more info on why the need to rename? I'm okay with either name, just trying to figure out which one should be used across.

Reply:
I probably should just remove one and make one handle both scenarios.  It was a quick and dirty way to handle the following routes.
`/api/admin/parcels/1231`
or
`/api/admin/parcels/000-001-231`

Reply:
I get the need for two routes as one is for an **[int]** and the other is a **[regex]**; however I was referring that the routes are **different**; i.e:

`/api/admin/parcels/pid/1231`
or
`/api/admin/parcels/p-id/000-001-231`

Notice one is `/pid/...` and the other is `/p-id` - I think you should pick either one of them and rename the other route to match

Reply:
They had the same path originally, but Swagger doesn't like it.